config/default/jenkins-infra.yaml

```diff
@@ -0,0 +1,127 @@
+                        - basicSSHUserPrivateKey:
+                            scope: GLOBAL
+                            id: "charts-secrets"
+                            username: ${SSH_CHARTS_SECRETS_USERNAME}
+                            #passphrase: ${SSH_CHARTS_SECRETS_PASSWORD}
+                            description: "SSH privkey used to access jenkins-infra/charts-secret"
+                            privateKeySource:
+                              directEntry:
+                                privateKey: ${SSH_CHARTS_SECRETS_PRIVKEY}
+              k8s-settings: |
+                jenkins:
+                  clouds:
+                    - kubernetes:
+                        containerCapStr: "100"
+                        jenkinsTunnel: "default-jenkins-infra-agent:50000"
+                        jenkinsUrl: "http://default-jenkins-infra:8080"
+                        maxRequestsPerHostStr: "300"
+                        name: "kubernetes"
+                        namespace: "release"
+                        podRetention: "Never"
+                        serverUrl: "https://kubernetes.default"
+                        templates:
+                          - name: jnlp-linux
+                            nodeSelector: "kubernetes.io/os=linux"
+                            containers:
+                              - name: jnlp
+                                image: "jenkins/jnlp-slave:latest-jdk11"
+                                resourceLimitCpu: "500m"
+                                resourceLimitMemory: "512Mi"
+                                resourceRequestCpu: "500m"
+                                resourceRequestMemory: "512Mi"
+                                args: "^${computer.jnlpmac} ^${computer.name}"
+                                alwaysPullImage: true
+                          - name: jnlp-windows
+                            nodeSelector: "kubernetes.io/os=windows"
+                            containers:
+                              - name: jnlp
+                                image: "jenkins4eval/jnlp-agent:latest-windows"
+                                resourceLimitCpu: "500m"
+                                resourceLimitMemory: "512Mi"
+                                resourceRequestCpu: "500m"
+                                resourceRequestMemory: "512Mi"
+                                args: "^${computer.jnlpmac} ^${computer.name}"
+                                alwaysPullImage: true
+                            label: "jnlp-windows"
+                            yaml: |-
+                              spec:
+                                tolerations:
+                                - key: "os"
+                                  operator: "Equal"
+                                  value: "windows"
+                                  effect: "NoSchedule"
+              jobs-settings: |
+                jobs:
+                  - script: >
+                      multibranchPipelineJob('k8smgmt') {
```

Comment:
do you plan to move some of the other infra jobs here as well? like azure? and plugin-site-api?

Reply:
I don't think so because that this service needs cluster-admin permission to manage the cluster, I want to keep it only the vpn and I wouldn't allow to many people with access on it.